name_that_hash/hashes.py

```diff
@@ -1641,4 +1641,59 @@ class HashInfo:
                 )
         ],
     ),
+    Prototype(
+    regex=re.compile(r"^(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,39}$", re.IGNORECASE),
```

Comment:
Where did you get that regex from? If you look at HashCat's example Bitcoin hash, it doesn't look like that: `$bitcoin$96$d011a1b6a8d675b7a36d0cd2efaca32a9f8dc1d57d6d01a58399ea04e703e8bbb44899039326f7a00f171a7bbc854a54$16$1563277210780230$158555$96$628835426818227243334570448571536352510740823233055715845322741625407685873076027233865346542174$66$625882875480513751851333441623702852811440775888122046360561760525`

Reply:
Solved but the regex is horrible:

```
\$bitcoin\$[0-9]{2,4}\$[a-fA-F0-9]{90,100}\$[0-9]{2,4}\$[a-fA-F0-9]{15,25}\$[0-9]{2,10}\$[0-9]{2,4}\$[a-fA-F0-9]{75,100}\$[0-9]{2,4}\$[a-fA-F0-9]{50,75}
```

Reply:
There's not enough examples for me to figure out the _exact_ numbers

Reply:
This can be solved with `\$bitcoin\$[0-9]{2,4}\$[a-fA-F0-9$]{200,350}`